CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
 Changes by Version
 ==================
 
-3.7.1 (unreleased)
+3.7.1 (2017-01-24)
 ------------------
 
-- nothing yet
+- UDP sender bug fixes (#184, #192, #195) 
```

Comment:
what specifically? Is it one bug or many? If many we need to break this line and describe each fix.